frontend/src/concepts/pipelines/content/pipelinesDetails/pipeline/PipelineDetails.tsx

```diff
@@ -73,10 +74,12 @@ const PipelineDetails: PipelineCoreDetailsPageComponent = ({ breadcrumbPath }) =
               breadcrumb={
                 <Breadcrumb>
                   {breadcrumbPath}
-                  <BreadcrumbItem isActive>{pipeline?.name || 'Loading...'}</BreadcrumbItem>
+                  <BreadcrumbItem isActive>
+                    <Truncate content={pipeline?.name || 'Loading...'} />
+                  </BreadcrumbItem>
```

Comment:
`PipelineRunDetails` uses a max width but not this one. 
We need to be consistent if that's the ask from UX.